packages/documentation-framework/scripts/md/parseMD.js

```diff
@@ -60,15 +60,17 @@ function toReactComponent(mdFilePath, source, buildMode) {
         sourceRepo = 'patternfly-react';
       }
 
-      const propComponents = [...new Set(frontmatter.propComponents || [])]
-        .filter(propComponent => {
-          if (tsDocs[propComponent]) {
-            return true;
-          }
-          file.message(`Prop component ${propComponent} missing from tsDocgen`);
-          return false;
-        })
-        .map(propComponent => tsDocs[propComponent])
+      const propComponents = [...new Set(frontmatter.propComponents || [])].reduce((acc, componentName) => {
+        const name = getTsDocName(componentName, source === 'react-next');
+
+        if (tsDocs[name]) {
```

Comment:
Trying to understand. So if there's let's say ...next/components/Wizard/Wizard.tsx, then it exists as `tsDocs['Wizard-next']`?
And then it will get pushed to 
```
propComponents[
  {
    "name": "Wizard-next",
    "props": []
  }
]
```
?

Reply:
No, the name of the component would be "Wizard" still, but the props would be that of the appropriate component.

Here's an example generated output for the "Next" Wizard:
```
const pageData = {
  "id": "Wizard",
  "section": "components",
  "source": "react-next",
  "slug": "/components/wizard/react-next",
  "sourceLink": "https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/next/components/Wizard/examples/Wizard.md",
  "propComponents": [
    {
      "name": "Wizard",
```